### ISSUE TITLE
use android as webcam?

### ISSUE THREAD

**Issue by anandijain** [teamclouday/AndroidMic#32]
Thanks for the amazing work! 
I was wondering how much additional work it would be to use the camera as an external webcam. It seems like it probably would use a lot of the same work

**Comment by teamclouday** [comment]
Yes, I was a contributor of this project called [OpenCamera](https://github.com/yukimuon/OpenCamera). The idea was to use Android phone camera as Windows pc camera. 

But there are limitations, mostly on the Windows side. The best way to simulate a camera from video stream on Windows is to write a driver. However, we weren't able to pay for a driver certificate. So instead we used an API called [DirectShow](https://learn.microsoft.com/en-us/windows/win32/directshow/directshow), which is legacy API and therefore only supported by a few softwares that use webcam.

**Comment by quadrinex** [comment]
> Yes, I was a contributor of this project called [OpenCamera](https://github.com/yukimuon/OpenCamera). The idea was to use Android phone camera as Windows pc camera.
> 
> But there are limitations, mostly on the Windows side. The best way to simulate a camera from video stream on Windows is to write a driver. However, we weren't able to pay for a driver certificate. So instead we used an API called [DirectShow](https://learn.microsoft.com/en-us/windows/win32/directshow/directshow), which is legacy API and therefore only supported by a few softwares that use webcam.

You could use obs webcam system for that (just put a window with the webcam or if you are motivated you could make a lite version of obs and keep the webcam (i think droidcam use that))
